README.md

```diff
@@ -99,6 +99,13 @@ simplify using those repositories.
 
 To install them, follow the instructions [here](INSTALLING_PACKAGES.md).
 
+
+## Building an ISO
+
+To build an ISO with the built packages, follow those
+[instructions](BUILDING_AN_ISO).
```

Comment:
Could you please remove this information from the CONTRIBUTING file? There's no need to duplicate instructions.